helix-view/src/editor.rs

```diff
@@ -140,10 +153,12 @@ impl Editor {
                 let (view, doc) = current!(self);
 
                 // initialize selection for view
-                let selection = doc
-                    .selections
-                    .entry(view.id)
-                    .or_insert_with(|| Selection::point(0));
+                let pos = if let Some((row, col)) = pos {
```

Comment:
I think for code like this it's easier to just `pos.unwrap_or_default()` (default ends up being `(0, 0)` and eliminate the two branches.